Sources/Kitura/bodyParser/MultiPartBodyParser.swift

```diff
@@ -19,24 +19,13 @@ import LoggerAPI
 
 class MultiPartBodyParser: BodyParserProtocol {
     let boundaryData: Data
-    let endBoundaryData: Data
-    let newLineData: Data
-    let endHeaderData: Data
+    let endBoundaryData = Data("--".utf8)
+    let newLineData = Data("\r\n".utf8)
+    let endHeaderData = Data("\r\n\r\n".utf8)
```

Comment:
could these be static?

Reply:
Yes but then they have to be referenced as `MultiPartBodyParser.endBoundaryData` everywhere which makes the diff bigger and the code uglier. What do you think?

Reply:
Decided leave this as-is for now.